::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Charalampos Kominos <charalampos.kominos@enea.com>
Date: Thu, 24 Aug 2017 19:28:14 +0200
Subject: [PATCH] aarch64: skip configuration of hugepages

The reclass model contains instructions on how to setup nova.conf.
Those instructions need to be overwritten for ARM.

Signed-off-by: Guillermo Herrero <Guillermo.Herrero@enea.com>
Signed-off-by: Charalampos Kominos <charalampos.kominos@enea.com>
---
 .../classes/cluster/baremetal-mcp-ocata-common/openstack_compute.yml  | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-common/openstack_compute.yml b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-common/openstack_compute.yml
index a78850f..ce67588 100644
--- a/mcp/reclass/classes/cluster/baremetal-mcp-ocata-common/openstack_compute.yml
+++ b/mcp/reclass/classes/cluster/baremetal-mcp-ocata-common/openstack_compute.yml
@@ -12,8 +12,8 @@ classes:
   - system.linux.storage.loopback
   - system.glusterfs.client.cluster
   - system.nova.compute.cluster
-  - system.nova.compute.nfv.hugepages
-  - system.nova.compute.nfv.cpu_pinning
+  # - system.nova.compute.nfv.hugepages
+  # - system.nova.compute.nfv.cpu_pinning
   - system.neutron.gateway.cluster
   - system.cinder.volume.single
   - system.cinder.volume.backend.lvm
